product_docs/docs/transporter/ba_preview/config_reader.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configuring"
+---
+
+## Config a source reader
+
+### Get credentials
+
+- Create and download a migration credentials from the postgres AI platform Migrate -> Credentials section
+- un-zip the credential and copy the whole folder to the host where the reader is installed
+```shell
+```
+
+### Configure the reader
+
+Set the following environment variables in /opt/cdcreader/run-cdcreader.sh with the right values:
+
+```shell
+### set the following environment variables:
+
+##########################################
+# Transporter Cloud Configuration        #
+##########################################
+
+# This ID is used to identify the cdcreader.
+#export DBZ_ID="my_reader_id"
+#export RW_SERVICE_HOST=https://transporter-rw-service.biganimal.com
```

Comment:
Do we need to replace something here or just uncomment?

Reply:
probably replace, I guess customers will not use the env var we are suggesting here, probably they'll prefer to set the abs path